extensions/markdown-language-features/src/languageFeatures/copyFiles/copyPaste.ts

```diff
@@ -27,8 +27,7 @@ class PasteEditProvider implements vscode.DocumentPasteEditProvider {
 			return createEdit;
 		}
 
-		const label = vscode.l10n.t('Insert Markdown Media');
-		const uriEdit = new vscode.DocumentPasteEdit('', this._id, label);
+		const uriEdit = new vscode.DocumentPasteEdit('', this._id, '');
```

Comment:
Do you also need to set the edit label below?

```ts
uriEdit.label = pasteEdit.label;
```

Reply:
Yes, I believe there should be a label for the final uriEdit